### ISSUE TITLE
Nargo in separate repository

### ISSUE THREAD

**Issue by Savio-Sou** [noir-lang/Planning#7]
Blocked by:
- [ ] Nargo Builds protected by Tests https://github.com/orgs/noir-lang/projects/1/views/9?pane=issue&itemId=21721855

**Comment by kobyhallx** [comment]
No longer relevant since we're pulling things into noir monorepo style
